src/core/index.ts

```diff
@@ -1,6 +1,5 @@
-import InstanceRigidBody from '../components/InstanceRigidBody.vue'
-import Physics from '../components/Physics.vue'
-import RigidBody from '../components/RigidBody.vue'
-import { useRapier } from './../composables/useRapier'
-
-export { InstanceRigidBody, Physics, RigidBody, useRapier }
+export * from '../components'
```

Comment:
I don't see the need for the core folder, if we follow the structure that we have now.

For information, the other pkg in the ecosystem have main logic inside this carpet (this includes components folders mostly) We need to discuss this, for the future could be better to have components folder inside core

Reply:
Yes, normally all the actual logic was supposed to be inside the `core` folder and the folder (like `utils`, `data`, `types`, etc), can be outside...

But we can do it in a separated PR

Reply:
Agree, another PR